### ISSUE TITLE
Create method for requesting ble permission

### ISSUE THREAD

**Issue by smstuebe** [xabre/MvvmCross-BluetoothLE#22]
Extend `IBluetoothLE` with `EnsurePermission()` and maybe `HasPermission`

**Comment by IlSocio** [comment]
Note that it does already exists the PermissionPlugin to handle all the logic related to permissions:
https://github.com/jamesmontemagno/PermissionsPlugin

**Comment by smstuebe** [comment]
@IlSocio Yes we know this. But we have to decide if we add a dependency on it or not and how :)
